en/config_rover/ackermann.md

```diff
@@ -102,41 +265,44 @@ This is done by scaling the acceptance radius based on the corner the rover has
-| -------------------------------------------------------------------------------------------------------- | ----------------------------------------------------- | ---- |
-| <a id="NAV_ACC_RAD"></a>[NAV_ACC_RAD](../advanced_config/parameter_reference.md#NAV_ACC_RAD)             | Default acceptance radius                             | m    |
-| <a id="RA_ACC_RAD_MAX"></a>[RA_ACC_RAD_MAX](../advanced_config/parameter_reference.md#RA_ACC_RAD_MAX)    | Maximum radius the acceptance radius can be scaled to | m    |
-| <a id="RA_ACC_RAD_GAIN"></a>[RA_ACC_RAD_GAIN](../advanced_config/parameter_reference.md#RA_ACC_RAD_GAIN) | Tuning parameter                                      | -    |
+::: info
+The corner cutting effect is a tradeoff between how close you get to the waypoint and the smoothness of the trajectory.
+:::
+
+1. [NAV_ACC_RAD](#NAV_ACC_RAD) [m]: Default acceptance radius. This is also used as a lower bound for the acceptance radius scaling.
+2. [RA_ACC_RAD_MAX](#RA_ACC_RAD_MAX) [m]: The maximum the acceptance radius can be scaled to. Set equal to [NAV_ACC_RAD](#NAV_ACC_RAD) to disable the corner cutting effect.
+3. [RA_ACC_RAD_GAIN](#RA_ACC_RAD_GAIN) [-]: This tuning parameter is a multiplicand on the [calculated ideal acceptance radius](#corner-cutting-logic) to account for dynamic effects.
 
-The tuning parameter is a multiplicand on the calculated ideal acceptance radius to account for dynamic effects.
+   :::tip
+   Initially set this parameter to `1`.
+   If you observe the rover overshooting the corner, increase this parameter until you are satisfied with the behaviour.
+   Note that the scaling of the acceptance radius is limited by [RA_ACC_RAD_MAX](#RA_ACC_RAD_MAX).
+   :::
 
-#### Corner slow down
+### Path Following
 
-To smoothen the trajectory further and reduce the risk of the rover rolling over, the mission speed is reduced as the rover gets closer to a waypoint:
+The [pure pursuit](#pure-pursuit-guidance-logic) algorithm is used to calculate a lateral acceleration setpoint for the vehicle that is then close loop controlled.
+The close loop lateral acceleration was tuned in the configuration of the [Acro mode](#acro-mode), and the pure pursuit was tuned when setting up the [Position mode](#position-mode).
+During any auto navigation task observe the behaviour of the rover.
 
-- During cornering the rover drives at a speed that is equal to the the inverse of the acceptance radius (calculated using the [corner cutting logic](#corner-cutting)) multiplied with a tuning parameter called [RA_MISS_VEL_GAIN](#RA_MISS_VEL_GAIN).
-- In between waypoints (straight line) the rover speed is regulated such that it will arrive at the acceptance radius of the waypoint with the desired cornering speed.
-  This requires [RA_MAX_ACCEL](#RA_MAX_ACCEL) and [RA_MAX_JERK](#RA_MAX_JERK) to be set.
+If you are unsatisfied with the path following, there are 3 steps to take:
```

Comment:
I see only two steps. Is it two, or has one been omitted?

Reply:
It is only 2 steps, i updated it accordingly (It was 3 steps for differential rovers and I initially copy-pasted that documenataion to keep the structure consistent and failed to adjust this correctly).